src/Pages/Event/Event.js

```diff
@@ -1,100 +1,197 @@
-        ,topImage:"/images/bg2.png",color:"#f27770"}
-        ,{id:"3",date:"03",month:"jan 23",topic:"Orientation Session",text:"We are having a live session where we will introduce you to Google Developer Students ClubsWe are having a live session where we will introduce you to Google Developer Students ClubsWe are having a live session where we will introduce you to Google Developer Students Clubs"
-        ,topImage:"/images/bg3.png",color:"#3eaff4"},
-        {id:"4",date:"04",month:"jan 23",topic:"Orientation Session",text:"We are having a live session where we will introduce you to Google Developer Students ClubsWe are having a live session where we will introduce you to Google Developer Students ClubsWe are having a live session where we will introduce you to Google Developer Students Clubs"
-        ,topImage:"/images/bg4.png",color:"#6ebb71"},
-        {id:"5",date:"06",month:"jan 23",topic:"Orientation Session",text:"We are having a live session where we will introduce you to Google Developer Students ClubsWe are having a live session where we will introduce you to Google Developer Students ClubsWe are having a live session where we will introduce you to Google Developer Students Clubs"
-        ,topImage:"/images/bg2.png",color:"#f47169"},
-        {id:"6",date:"06",month:"jan 23",topic:"Orientation Session",text:"We are having a live session where we will introduce you to Google Developer Students ClubsWe are having a live session where we will introduce you to Google Developer Students ClubsWe are having a live session where we will introduce you to Google Developer Students Clubs"
-        ,topImage:"/images/bg1.png",color:"#ffd34e"}
-    ])
-    const allEvents = upcomingdata.concat(pastData) 
-    /* allEvents is an array which consists of both upcoming and past events */
-    const [allData, setAllData] = useState(allEvents)
+  const [upcomingdata, setUpcomingData] = useState([
+    {
+      id: "1",
+      date: "11",
+      month: "jan 23",
+      topic: "Orientation Session",
+      text: "We are having a live session where we will introduce you to Google Developer Students ClubsWe are having a live session where we will introduce you to Google Developer Students ClubsWe are having a live session where we will introduce you to Google Developer Students Clubs",
+      topImage: "/images/bg1.png",
+      color: "#ffd34e",
+    },
+    {
+      id: "2",
+      date: "12",
+      month: "jan 23",
+      topic: "Orientation Session",
+      text: "We are having a live session where we will introduce you to Google Developer Students ClubsWe are having a live session where we will introduce you to Google Developer Students ClubsWe are having a live session where we will introduce you to Google Developer Students Clubs",
+      topImage: "/images/bg2.png",
+      color: "#f27770",
+    },
```

Comment:
@kamalchakma1 that's what I am talking about although I thought it was hard coded but we can move this to sperate file create issue for this it will be helpful

Reply:
ok i got your point. let me create an issue on this.